gui/cppchecklibrarydata.cpp

```diff
@@ -719,7 +721,7 @@ static void writeMemoryResource(QXmlStreamWriter &xmlWriter, const CppcheckLibra
         } else {
             xmlWriter.writeStartElement("alloc");
         }
-        xmlWriter.writeAttribute("init", alloc.init ? "true" : "false");
+        xmlWriter.writeAttribute("init", QString::fromStdString(bool_to_string(alloc.init)));
```

Comment:
redundant to use QString::fromStdString here.

Reply:
Done.